webapp/models.py

```diff
@@ -278,3 +279,41 @@ def _get_topic(self, path):
             raise error
 
         return response.json()
+
+
+def get_search_results(
+    api_key,
+    api_url,
+    search_custom_id,
+    query,
+    start,
+    num,
+    site=None,
+    session=SEARCH_SESSION,
+):
+    """
+    Query the Google Custom Search API for search results
+    """
+    if not api_key:
+        raise Exception("Unable to search: No API key provided")
+
+    results = session.get(
+        api_url,
+        params={
+            "key": api_key,
+            "cx": search_custom_id,
+            "q": query,
+            "siteSearch": site,
```

Comment:
As mentioned above, could we hardcode this to `"siteSearch": "docs.snapcraft.io"` and remove the parameter from this function?

Reply:
Hardcoded in the function now.